app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -64,6 +61,54 @@ def install_commands(version, install_dir)
           end
         end
 
+        def translate_docker_build(config)
+          [
+            ("# Attaching workspace at #{config['attach-at']}" if config['attach-at']),
+            ('export DOCKER_BUILDKIT=1' if config['use-buildkit']),
+            lint_dockerfile(config),
+            *pull_cache_images(config),
+            generate_build_command(config)
+          ].compact
+        end
+
+        def lint_dockerfile(config)
+          return unless config['lint-dockerfile']
+
+          "# Linting Dockerfile#{' (treating warnings as errors)' if config['treat-warnings-as-errors']}"
+        end
```

Comment:
3 comments about this function:
1- it is misleading because it apparently deals with 2 options not just one
2- why not just inline this function that just adds a single comment?
3- (same as with `atttach-at`) what do the options actually do?